File: ServerAndAngular/Dto/PackageWithProjectDto.cs
using SoupDiscover.ORM;

namespace SoupDiscover.Dto
{
    public class PackageWithProjectDto
    {
        public PackageWithProjectDto(Package packageDto, PackageConsumerDto[] packageConsumers)
        {
            this.packageDto = packageDto;
            this.packageConsumers = packageConsumers;
        }

        public Package packageDto { get; }

        public PackageConsumerDto[] packageConsumers { get; }
    }
}

File: ServerAndAngular/ClientApp/src/app/model/packageWithProjectDto.ts
import { PackageDto } from "./package";
import { PackageConsumerDto } from "./PackageConsumerDto";

export class PackageWithProjectDto {
  /**
   * The package
   * */
  packageDto: PackageDto;

  /*
   * All project name where is found the package
   */
  packageConsumers: PackageConsumerDto[];
}
